axlearn/common/ssm.py

```diff
@@ -971,6 +996,30 @@ def extend_step(
 class JambaMixerLayer(MambaMixerLayer):
     """A Jamba-style Mamba layer, which norms the input-dependent SSM parameters."""
 
+    def _ssm_parameters(self, inputs: Tensor) -> MambaMixerLayer.SSMParameters:
```

Comment:
This function also seems to duplicate the function originally defined below, on line 1036 (originally 987).

Reply:
You’re right. I’ll remove the duplicate function on line 1036. Thanks.